File: Jottings/Jotting+CoreDataProperties.m
//
//  Jotting+CoreDataProperties.m
//  Jottings
//
//  Created by Morten Kals on 01/09/2016.
//  Copyright © 2016 Kals. All rights reserved.
//

#import "Jotting+CoreDataProperties.h"

@implementation Jotting (CoreDataProperties)

+ (NSFetchRequest<Jotting *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Jotting"];
}

@dynamic lastChanged;
@dynamic author;
@dynamic versions;

@end

File: Jottings/Jotting+CoreDataProperties.swift
//
//  Jotting+CoreDataProperties.swift
//  Jottings
//
//  Created by Morten Kals on 29/08/2016.
//  Copyright © 2016 Kals. All rights reserved.
//
//  Choose "Create NSManagedObject Subclass…" from the Core Data editor menu
//  to delete and recreate this implementation file for your updated model.
//

import Foundation
import CoreData

extension Jotting {

    @NSManaged var author: Authour?
    @NSManaged var versions: NSSet?

}
